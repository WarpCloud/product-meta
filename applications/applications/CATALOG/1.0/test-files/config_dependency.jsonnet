{
  application_type: "CATALOG",
  application_name: "catalog",
  application_version: "1.0",
  user_config: {},
}
