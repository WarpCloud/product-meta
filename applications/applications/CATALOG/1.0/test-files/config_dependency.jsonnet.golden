{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "catalog",
            "application_type": "CATALOG"
         },
         "configs": {
            "catalog_platform_cpu_limit": 1,
            "catalog_platform_cpu_request": 1,
            "catalog_platform_memory_limit": 4,
            "catalog_platform_memory_request": 0.10000000000000001,
            "catalog_web_cpu_limit": 1,
            "catalog_web_cpu_request": 1,
            "catalog_web_memory_limit": 4,
            "catalog_web_memory_request": 0.10000000000000001
         },
         "dependencies": [
            {
               "moduleName": "txsql",
               "name": "txsql"
            },
            {
               "moduleName": "hyperbase",
               "name": "hyperbase"
            },
            {
               "moduleName": "kafka",
               "name": "kafka"
            },
            {
               "moduleName": "elasticsearch",
               "name": "elasticsearch"
            }
         ],
         "instance_settings": { },
         "moduleName": "catalog",
         "name": "catalog-catalog",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
